v1/terraform/Cloud_PubSub_to_Datadog/dataflow_job.tf

```diff
@@ -65,7 +65,7 @@ variable "parallelism" {
 
 variable "includePubsubMessage" {
   type        = bool
-  description = "Include full Pub/Sub message in the payload (true/false). Defaults to false (only data element is included in the payload)."
+  description = "Include full Pub/Sub message in the payload (true/false). Defaults to true (all elements, including data element, are included in the payload)."
   default     = null
```

Comment:
Should we update the default here as well?

Reply:
Hey @camilledolle apologies for the delay here, I specifically chose to not update the default here since we already have a similar setup with batchCount, parallelism and other fields in our terraform that instead have the default set in the template code. But additionally, would want this default change to occur as a change to their template not as an update to their terraform if possible. Happy to discuss if you disagree.